doc/src/faq.rst

```diff
@@ -228,10 +228,11 @@ line of the main module. For example:
         freeze_support()
         Process(target=f).start()
 
-If the freeze_support() line is omitted then trying to run the frozen
-executable will raise RuntimeError.
+If the freeze_support() line is omitted, then running the frozen executable
+will raise RuntimeError on Windows. On Linux and macOS a similar message is
+shown but cx_Freeze tries to run the program by injecting a freeze_support.
 
-Calling freeze_support() has no effect when invoked on any operating system
-other than Windows. In addition, if the module is being run normally by the
-Python interpreter on Windows (the program has not been frozen), then
-freeze_support() has no effect.
+Contrary to what is stated in the Python documentation, freeze_support() is
```

Comment:
I think this could be clarified a bit. The docs (python) are a bit unclear because the need to cause freezesupoort depends on the method to create new processes which widely varies by platform.  Imo I’d mention the freeze from spawn and from base multiprocessing here even if handled by the hook

Reply:
What was done in the hook was to normalize the use of OS-independent freeze_support (monkey-patching it with spaw.freeze_support) and I think it would confuse some people as it would seem like double information or a choice between them depending on the method, with in any case, spaw.freeze_support will always be used.

Reply:
Fair enough, is there one needed for the third type of processing: forkserver

Reply:
There's not

Reply:
It reuses the spawn one if I'm reading correctly

Reply:
It works for forkserver, due to an error I didn't enable the test for forkserver.

Reply:
```suggestion
Contrary to what the Python docs may state, you MUST use `multiprocessing.freeze_support()` on MacOS, Windows, and Linux. On Mac and Linux, cx_Freeze patches the call to also handle `multiprocessing.spawn.freeze_support()` when needed.
```

Reply:
This isn't my best work of literature, but I think it is more clear what the patch is doing in case someone stumbles across it later

Reply:
Due to the characteristics of forkserver, only the sample3 test can be used with it.
The test pass: https://github.com/marcelotduarte/cx_Freeze/actions/runs/9411907451/job/25925953665?pr=2443#step:6:286